apps/website/blog/007-migrating-to-pnpm.mdx

```diff
@@ -0,0 +1,276 @@
+---
+title: "Migrating to PNPM: What's Next for the PagoPA DX Toolchain"
+slug: migrating-to-pnpm
+hide_table_of_contents: false
+date: 2025-10-03T08:00
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+draft: true
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Big news for the PagoPA developer community! 🎉
+
+We're officially moving from Yarn to
+[PNPM](https://pagopa.github.io/technology-radar/tools/pnpm.html) for our
+dependency management. This change is a significant step forward for the PagoPA
+DX Toolchain, bringing improvements in speed, consistency and security to our
+development workflow.
+
+:::warning Deprecation notice We understand that a change of this magnitude
```

Comment:
issue: fix formatting

Reply:
Fixed, we need to find a better configuration for `prettier` on docusaurus. It is the same problem of the devcontainer page.